### ISSUE TITLE
Title text is empty when window is fullscreenized with vertical tab

### ISSUE THREAD

**Issue by simonhong** [brave/brave-browser#34474]
## Steps to Reproduce

   1. Enable vertical tab and show title bar options
   2. Maximize browser window
   3. 

## Actual result:

<img width="2191" alt="image" src="https://github.com/brave/brave-browser/assets/6786187/9863c9ca-e69d-4084-ba7b-3353b8ca06ed">

## Expected result:
title bar should have title text

## Reproduces how often: 

Easy

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:

cc @rebron @sangwoo108

**Comment by simonhong** [comment]
Or maybe we want to hide titlebar always whenever goes into maximized?

**Comment by sangwoo108** [comment]
Yeah, if we're talking about the Mac's "Zoom" button behavior, title is hidden from the client view. I think we should adjust the insets, bc the insets are wrong when the "Show title" option is off too. 

<img width="955" alt="image" src="https://github.com/brave/brave-browser/assets/5474642/4daa00c8-446c-4a57-bdd0-b64ad66a1de1">

**Comment by bsclifton** [comment]
This has been resolved! 😄

**Comment by bsclifton** [comment]
Whoops - I missed that this was macOS. Re-opening
